README.md

```diff
@@ -84,14 +84,29 @@ Due to [network limitation](https://docs.docker.com/docker-for-mac/networking/#k
 $ EXPOSE_HOST_PORTS=true make docker-testacc
 ```
 
+Branching the Provider
+---------------------------
+
+The provider is branched into two release lines that have major version alignment with Rancher 2.6 and 2.7. The `release/v2` branch with 2.0.0+ is aligned with Rancher 2.6 and `master` with 3.0.0+ is aligned with Rancher 2.7. Terraform fixes and new features will be available on `master` but only bug fixes will be backported to `release/v2.`
```

Comment:
Don't want to necessarily commit to backporting all fixes, so adding a bit of a disclaimer. Also minor formatting fix.

```suggestion
The provider is branched into two release lines that have major version alignment with Rancher 2.6 and 2.7. The `release/v2` branch with 2.0.0+ is aligned with Rancher 2.6 and `master` with 3.0.0+ is aligned with Rancher 2.7. Terraform provider fixes and new features will be available on `master` but only bug fixes will be backported to `release/v2` as necessary.
```